pkg/autoscaler/aggregation/bucketing.go

```diff
@@ -52,12 +52,12 @@ func (t *TimedFloat64Buckets) Record(time time.Time, name string, value float64)
 	bucket.Record(name, value)
 }
 
-// IsEmpty returns whether or not there are no values currently stored.
-func (t *TimedFloat64Buckets) IsEmpty() bool {
+// Size returns the number of buckets stored.
+func (t *TimedFloat64Buckets) Size() int {
```

Comment:
is this used outside tests? I.e. should it be package private and probably in the `_test` file?

Reply:
Ditched the method and replaced in tests accordingly.